### PR TITLE
[BLCKCHN-194] Create python example for agent wallet signer #14

### PR DESCRIPTION
**Pull request by leejw51crypto**
add python example for wallet signer

### CODE REVIEW

File: ai/cryptocom-ai-agent-pychatbot/chat.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import os
+import requests
+import json
+import webbrowser
+from dotenv import load_dotenv
+
+# Load environment variables from .env file
```
Comment (rarcifa) [inline ai/cryptocom-ai-agent-pychatbot/chat.py:7]: can be removed

**Review by rarcifa — APPROVED** [review]

**Comment by rarcifa** [comment]
@leejw51crypto maybe lets also add some comprehensive Readme file, to explain what the pre requirements are (e.g. service running, etc.)

**Review by yeehan-crypto-com — APPROVED** [review]
LGTM, i think we can also add more help to setting up the project:
- requirements.txt file
- .env.example file

**Comment by leejw51crypto** [comment]
ok, i'll add readme, requirements.txt, .env.example

**Pull request merged**